webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -150,166 +147,145 @@ class ManageManifestModal extends Component {
-                          />
-                        </Col>
-                      </FormGroup>
-                    </React.Fragment>
-                  }
-                </Form>
-              </Tab>
-            }
-            <Tab eventKey={2} title={__('Manifest History')}>
-              <LoadingState loading={manifestHistory.loading} loadingText={__('Loading')}>
-                <Table
-                  rows={manifestHistory.results}
-                  columns={columns}
-                  emptyState={emptyStateData()}
-                />
-              </LoadingState>
+                          {canDeleteManifest &&
+                          <React.Fragment>
+                            <TooltipButton
+                              renderedButton={(
+                                <Button
+                                  disabled={!manifestExists(organization) || actionInProgress}
+                                  bsStyle="danger"
+                                  onClick={this.showDeleteManifestModal}
+                                >
+                                  {__('Delete')}
+                                </Button>
+                                )}
+
+                              tooltipId="delete-manifest-button-tooltip"
+                              tooltipText={this.disabledTooltipText()}
+                              tooltipPlacement="top"
+
+                            />
+                          </React.Fragment>
+                          }
+                        </div>
+                        <ForemanModal title={__('Confirm delete manifest')} id={DELETE_MANIFEST_MODAL_ID}>
+                          <DeleteManifestModalText />
+                          <ForemanModal.Footer>
+                            <Button bsStyle="default" onClick={this.hideDeleteManifestModal}>
+                              {__('Cancel')}
+                            </Button>
+                            <Button bsStyle="danger" onClick={() => this.deleteManifest()}>
+                              {__('Delete')}
+                            </Button>
+                          </ForemanModal.Footer>
+                        </ForemanModal>
+                      </Col>
+                    </FormGroup>
+                  </React.Fragment>
+                }
+              </Form>
             </Tab>
-          </Tabs>
-        </Modal.Body>
-        <Modal.Footer>
+          }
+          <Tab eventKey={2} title={__('Manifest History')}>
+            <LoadingState loading={manifestHistory.loading} loadingText={__('Loading')}>
+              <Table
+                rows={manifestHistory.results}
+                columns={columns}
+                emptyState={emptyStateData()}
+              />
+            </LoadingState>
+          </Tab>
+        </Tabs>
+        <ForemanModal.Footer>
           <Button bsStyle="primary" onClick={this.hideModal}>
```

Comment:
I'm getting an error here when trying to close the modal:
`ForemanModalActions.js:35 Uncaught Error: SET_MODAL_CLOSED error: Modal with id 'deleteManifestModal' does not exist`

Reply:
@MariaAga I'm not seeing this error locally, do you remember what steps you were going through to reproduce it?

Reply:
https://github.com/Katello/katello/blob/db28f63630d19f6371a1122f6441d2afce39db9d/webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js#L110

You can change `showSubscriptionManifest` to false to reproduce it

Reply:
@MariaAga Thanks, updated.  I may do a PR in Foreman to not throw this error anyway.  It feels like it's too much, and since we use constants for modal ID anyway, I feel like that's enough of a guard against using the wrong modal ID when managing your modals.

Reply:
Great!